pkg/cli/config/config.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/openshift/backplane-cli/pkg/info"
+)
+
+type BackplaneConfiguration struct {
+	URL       string
+	Proxy_URL string
+}
+
+func GetBackplaneConfigFile() string {
+	path, bpConfigFound := os.LookupEnv(info.BACKPLANE_CONFIG_PATH_ENV_NAME)
+	if bpConfigFound {
+		return path
+	}
+
+	return getConfiDefaultPath(info.BACKPLANE_CONFIG_DEFAULT_FILE_NAME)
+}
+
+// Get Backplane config default path
+func getConfiDefaultPath(fileName string) string {
+	configDir, err := os.UserConfigDir()
+
+	if err != nil {
+		return fileName
+	}
+
+	return configDir + "/" + fileName
+}
+
+// Get Backplane ProxyUrl from config
+func GetBackplaneProxyUrl() (proxyUrl string, err error) {
```

Comment:
Rename this to GetBackplaneConfiguration  and return the config struct dirrectly?